src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Annotations/Annotation.cs

```diff
@@ -634,27 +636,29 @@ private void ReadAttributes(XmlReader reader)
                     #pragma warning restore 0618
 
                     case AnnotationXmlConstants.Attributes.TypeName:
-                        string[] typeName = value.Split(_Colon);
-                        if (typeName.Length == 1)
+                        ReadOnlySpan<char> typeName = value.AsSpan();
+                        int segmentsLength = typeName.Split(segments, Colon);
```

Comment:
```suggestion
                        int segmentsLength = typeName.Split(segments, Colon, StringSplitOptions.TrimEntries);
```
and remove all subsequent calls to `Trim`?

Reply:
Yeah, why not, should be even faster if there's an actual need to trim, which shouldn't be the case though.